universal-application-tool-0.0.1/app/services/cloud/azure/BlobStorageUploadRequest.java

```diff
@@ -0,0 +1,89 @@
+package services.cloud.azure;
+
+import com.google.auto.value.AutoValue;
+import services.cloud.StorageServiceName;
+import services.cloud.StorageUploadRequest;
+
+/** This holds the information needed to upload a blob storage request. */
```

Comment:
nit: is it for uploading a request or making a request to upload a file to blob storage?